doc/panes/editor.rst

```diff
@@ -2,22 +2,41 @@
 Editor
 ######
 
-Spyder's multi-language **Editor** integrates a number of powerful tools right out of the box for an easy to use, efficient editing experience.
-The Editor's key features include syntax highlighting (``pygments``); real-time code and style analysis (``pyflakes`` and ``pycodestyle``); on-demand completion, calltips and go-to-definition features (``rope`` and ``jedi``); a function/class browser, horizontal and vertical splitting, and much more.
+Spyder's multi-language **Editor** pane is the key element of the IDE where users can create, open, and modify code files.
+The Editor offers a variety of core features like autocompletion, real-time analysis, syntax highlighting, horizontal and vertical splitting, and much more.
+In addition, it uniquely integrates a number of powerful tools for an easy to use, efficient editing experience.
 
 .. image:: /images/editor/editor-standard.png
    :alt: Spyder's Editor panel, split horizontally and with style analysis
 
-:guilabel:`Outline Explorer` (function/class/method browser) and horizontal/vertical splitting capabilities:
+===========================
+The Editor's key components
+===========================
 
-.. image:: /images/editor/editor-outline-standard.png
-   :alt: Spyder outline panel, showing the functions/classes/methods in a file
+The editor pane consists of the following areas:
 
-Real-time code and style analysis with ``pyflakes`` and ``pycodestyle``:
+.. image:: /images/editor/editor-pane.png
+   :alt: Spyder's Editor panel, showing the different areas of the editor
 
-.. image:: /images/editor/editor-inset-code-analysis.png
-   :alt: A snippit of code in the Spyder Editor, showing code style warnings
 
+1. The left sidebar shows line numbers and displays any code analysis errors/warnings that exist in the current file.
+   Clicking a line number highlights the code in that line.
+
+.. image:: /images/editor/editor-pane-code-error.png
+   :alt: Spyder outline panel, showing an example of a code error
+
+2. The right and bottom scrollbars help navigate the code in the current file vertically and horizontally.
+3. The Context menu displays actions relevant to the editor pane. To display the editor's context menu, right-click anywhere over it and a list of possible options will appear under the cursor.
+4. Another menu called the Options menu appears when clicking the "Hamburger" icon at the top right, which includes useful settings and actions relevant to the editor.
+
+.. image:: /images/editor/editor-pane-options-menu.png
+   :alt: Spyder outline panel with the options menu expanded
+
```

Comment:
```suggestion
```

At least per the current convention used on all the other panes that have substantial Options menus, we have a separate section specifically describing the functionality, where this screenshot would likely be a better fit, since it will be needed there anyway. Therefore, I would suggest saving it for that section and linking it here once you add it, which would also avoid it breaking up the flow of the numbered list of components and would make them shorter, where it is easier for readers to see both the image and the annotations and thus easier to understand and follow.

Also, for when this is used, it should be cropped to focus just on the outline menu, like all the similar screenshots in the other panes' docs are. This takes up much less space on screen and in the file, helps the reader focus on what's important and makes it much easier to read; as it stands now, I find it very hard to read the text.